### PR TITLE
Data update

### PR DESCRIPTION
**Pull request by NinaWie**
Update of the database including Avi's data and website donations. Also updated the readme and the dataset metadata sheet accordingly. Additionally some cleanup of unnecessary folders in the data folder.
We might want to wait until the comments of our MDs about the website donation data (in the google sheet), everything else is fine to merge.

### CODE REVIEW

File: pocovidnet/pocovidnet/utils_butterfly_data.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
 
 def get_paths(data_table_path):
     """
     Get the filenames of our data from csv file
     """
-    data_table = pd.read_csv(data_table_path)
-    butterfly = data_table["Filename"].values
-    butterfly = [b for b in butterfly if "utterfly" in b]
+    with open(data_table_path, "r") as infile:
```
Comment (jannisborn) [inline pocovidnet/pocovidnet/utils_butterfly_data.py:9]: Are you sure that this does not break the execution of `parse_butterfly.sh`?

File: pocovidnet/scripts/eval_vid_classifier.py [hunk 2]
```diff
@@ -11,9 +11,7 @@
 
 def main():
     parser = argparse.ArgumentParser(description='Evaluate genesis and cam')
-    parser.add_argument(
-        '--json', type=str, default="../data/video_input_data/cross_val.json"
-    )
+    parser.add_argument('--json', type=str, default="../data/cross_val.json")
```
Comment (jannisborn) [inline pocovidnet/scripts/eval_vid_classifier.py:14]: I checked the `cross_val.json`. Are we implicitly changing here to a json that has a fixed mapping between each sample and the fold? If so, what do we do if we add more data?

File: data/README.md [hunk 3]
```diff
@@ -19,17 +19,20 @@ assembled from publicly available resources in the web as well as from publicati
...
-- Linear: 
-  - 21 videos (4x COVID, 2x bacterial pneumonia, 11x healthy, 4x viral pneumonia)
-  - 5 images (3x COVID, 2x bacterial pneumonia)
+  - 108 videos (33x  COVID, 23x bacterial pneumonia, 49x healthy, 3x viral pneumonia)
+  - 33 images (18x COVID, 7x bacterial pneumonia, 8x healthy)
+  - 21 videos from the Butterfly dataset (19 COVID, 2 healthy, see below how to use the provided scripts to process the data)
+- Linear:
+  - 20 videos (4x COVID, 2x bacterial pneumonia, 10x healthy, 4x viral pneumonia)
+  - 6 images (4x COVID, 2x bacterial pneumonia)
+- Data with artifacts (30 images, 2 videos): For the sake of completion, we provide a folder with images and videos that we found but did not include in our dataset due to artifacts (arrows, bars) in the image or because of unclear labels. For more information and comments by medical experts, see our [metadata](https://docs.google.com/spreadsheets/d/1t-tLMjMod6W-nAjkuxmO0CLsiyalFIOp92k_XD_yeo8/edit#gid=1181682638) sheet or metadata [csv](dataset_metadata.csv).
```
Comment (jannisborn) [inline data/README.md:30]: Why is the new dataset much smaller than the old one? I thought most of the data with artifacts was just added freshly to the DB. Was anything moved from clean to artifact data, or how did we loose samples?

File: pocovidnet/pocovidnet/utils_butterfly_data.py (hunk 1 above)
Comment (NinaWie) [reply pocovidnet/pocovidnet/utils_butterfly_data.py:9]: Yes, I tested it, it is changed in both butterfly scripts

**Pull request merged**